setup/index.md

```diff
@@ -4,22 +4,36 @@ title: Testing Setup
 root: ..
 ---
 
-This directory contains scripts for testing your machine to make sure
+This directory contains scripts for installing the 
+environment needed for the course and testing your machine to make sure
 you have the software you'll need for your workshop installed.  To use
 these scripts:
 
-1.  Download [swc-installation-test-1.py](swc-installation-test-1.py).
+1.  Download [swc-installation.sh](swc-installation.sh).
```

Comment:
This won't work unless they have got bash and conda link with it. I.e., windows users won't be able to run this script.

Reply:
Yes, I was thinking that they would run this via the terminal in the Jupyter after they had installed Anaconda. Would that work (and if so, I guess I should include that in the instructions!)

Reply:
The powershell addition and changes to Window are really helpful, thanks!

Reply:
I've created a powershell script for windows. I tested and it works either running it on the powershell or by clicking.